src/main/features/claim/routes/submitted.ts

```diff
@@ -1,24 +1,27 @@
 import * as express from 'express'
 import { Paths } from 'claim/paths'
 import ErrorHandling from 'common/errorHandling'
+import Claim from 'claims/models/claim'
+import ClaimStoreClient from 'claims/claimStoreClient'
 
-function renderView (req: express.Request, res: express.Response, next: express.NextFunction): void {
-  // TODO: The below values needs to be retrieved from claim store, should be done as part of/after ROC-1796
-  const today = new Date()
+async function renderView (req: express.Request, res: express.Response): Promise<void> {
+  const { externalId } = req.params
+
+  const claim: Claim = await ClaimStoreClient.retrieveByExternalId(externalId)
   res.render(Paths.claimSubmittedPage.associatedView, {
-    claimNumber: 'D99YJ987',
-    submittedDate: today,
-    issueDate: today,
-    feePaid: '70',
-    repEmail: 'test@email.com',
-    externalId: 'xxxxxx'
+    claimNumber: claim.claimNumber,
+    submittedDate: claim.createdAt,
+    issueDate: claim.issuedOn,
+    feePaid: claim.claimData.paidFeeAmount,
+    repEmail: claim.claimantEmail,
```

Comment:
This should be idam email, which is user id (legal rep).

Reply:
it is storing Idam email address based on user authorization passed to claim store usin